openomni/bin/omni_listen_rfcat

```diff
@@ -27,34 +33,55 @@ def main(options=None):
 
     d = RfCat(0, debug=False)
     openomni.rf.configure_rfcat(d)
-    json = args.json
-
-    while not keystop():
-        try:
-            pkt, ts = d.RFrecv(timeout=80000)
-            pkt = Packet.flip_bytes(pkt)
-            rcv_time = datetime.datetime.now()
-
-            x = 0
-            while x < len(pkt):
-                data = pkt[:len(pkt) - (x + 1)]
-                packet = Packet(data)
-                packet.received_at = rcv_time
-                if packet.is_valid():
-                    #print(data.encode("hex"))
-                    if args.raw:
-                        print(packet.data.encode("hex"))
-                    if json:
-                        print(packet.as_json())
-                    else:
-                        print(packet)
-                    break
-                x += 1
-
-        except ChipconUsbTimeoutException:
-            time.sleep(0.5)
-
-    sys.stdin.read(1)
+
+    # Default variables for filename
+    dateRecording = datetime.today().isoformat()
+    userName = getpass.getuser()
+    fileName = '%s-%s' % (dateRecording, userName)
```

Comment:
I am also looking for a better way of naming the files. Now there only date and username, so finding your results back is a bit cumbersome. The free text input would be a bit to unorganised as a filename I think. Do you have any suggestions?